Mathlib/RepresentationTheory/Homological/GroupCohomology/Basic.lean

```diff
@@ -129,6 +129,7 @@ which calculates the group cohomology of `A`. -/
 noncomputable abbrev inhomogeneousCochains : CochainComplex (ModuleCat k) ℕ :=
   CochainComplex.of (fun n => ModuleCat.of k ((Fin n → G) → A))
     (fun n => inhomogeneousCochains.d A n) fun n => by
+    classical
```

Comment:
This is the fix, everything else is just responding to warnings about unused arguments.